services/121-service/src/payments/payments.module.ts

```diff
@@ -71,4 +74,25 @@ import { createScopedRepositoryProvider } from '@121-service/src/utils/scope/cre
     PaymentsProgressHelperService,
   ],
 })
-export class PaymentsModule {}
+export class PaymentsModule implements OnModuleInit {
+  private readonly logger = new Logger(PaymentsModule.name);
```

Comment:
this is not something we're using anywhere else already right? What is the added value of deviating from the standard?